semantics/llvm-syntax.k

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2013-2014 K Team. All Rights Reserved.
+/*@
+
+This is the syntax of LLVM.
+
+This syntax requires a input LLVM program without any comments on it.
```

Comment:
Besides the English, I have no idea what you mean.  Are you trying to say that comments are not supported?  Why not?  Also, the syntax does not "require" any program.  Please clarify what you mean.